### PR TITLE
Helium 2.7.1

### PR DESCRIPTION
**Pull request by cb-jpadilla**
Helium 2.7.1

### CODE REVIEW

File: HeliumExample/build.gradle [hunk 1]
```diff
@@ -22,42 +23,41 @@ android {
...
+    implementation 'com.chartboost:helium-applovin:2.7.1.0'
+    implementation 'com.chartboost:helium-unityads:2.7.1.0'
+    implementation 'com.chartboost:helium-ironsource:2.7.1.0'
+    implementation 'com.chartboost:helium-fyber:2.7.1.0'
+    implementation 'com.chartboost:helium-inmobi:2.7.1.0'
+    implementation 'com.chartboost:helium-mintegral:2.7.1.0'
+
     //Tapjoy
-    implementation 'com.tapjoy:tapjoy-android-sdk:12.8.1'
+    implementation 'com.tapjoy:tapjoy-android-sdk:12.9.0'
```
Comment (bichenwang) [inline HeliumExample/build.gradle:60]: are these not automatically pulled in via the above adapters?

File: README.md [hunk 2]
```diff
@@ -219,7 +263,7 @@ Note: Not all partner SDKs have full support for GDPR. Please refer to its offic
 
 ## 3rd-Party Supported Partner SDKs & Adapters
 ---
-As of 2.0.0, the Helium SDK currently supports the following 3rd-party programmatic & mediated partner sdks:
+As of 2.7.0, the Helium SDK currently supports the following 3rd-party programmatic & mediated partner sdks:
```
Comment (bichenwang) [inline README.md:266]: 2.7.1

File: HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt [hunk 3]
```diff
@@ -152,6 +153,9 @@ class MainActivity : AppCompatActivity() {
                     if (error != null) {
                         addToLogView(placementName + "(HeliumInterstitialAd) didShow failed with error: " + error.getMessage())
                     } else {
+                        Handler(Looper.getMainLooper()).post {
+                            binding.btnLoad.isEnabled = true
```
Comment (bichenwang) [inline HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt:157]: I'm actually not a fan of blocking load buttons in general.

File: HeliumExample/build.gradle (hunk 1 above)
Comment (cb-jpadilla) [reply HeliumExample/build.gradle:60]: Yes, but this helps check partner version we currently support for reference.

File: HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt (hunk 3 above)
Comment (cb-jpadilla) [reply HeliumExample/src/main/java/com/chartboost/heliumsdk/sampleapp/MainActivity.kt:157]: Can make new adjustments here as well.

**Review by bichenwang — APPROVED** [review]
lgtm

**Pull request merged**